modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/checkpoint/CheckpointManager.java

```diff
@@ -203,9 +204,7 @@ public CheckpointManager(
 
         checkpointer = checkpointerProvider.get();
 
-        Long cfgCheckpointReadLockTimeout = persistenceCfg != null
-            ? persistenceCfg.getCheckpointReadLockTimeout()
-            : null;
+        Long cfgCheckpointReadLockTimeout = persistenceCfg.getCheckpointReadLockTimeout();
```

Comment:
We call methods of `persistenceCfg` multiple times in this constructor before reaching line 207, that's why null check is redundant here